src/main/scala/com/github/pjfanning/pekkobuild/PekkoDevelocityPlugin.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.github.pjfanning.pekkobuild
+
+import com.gradle.develocity.agent.sbt.DevelocityPlugin
+import com.gradle.develocity.agent.sbt.DevelocityPlugin.autoImport.{
+  develocityConfiguration,
+  FlakyTestPolicy,
+  ProjectId,
+  Publishing
+}
+import sbt.{AutoPlugin, Plugins, PluginTrigger, Setting}
+import sbt.Keys.insideCI
+
+import java.net.URI
+
+object PekkoDevelocityPlugin extends AutoPlugin {
+
+  private val ApacheDevelocityUrl   = URI.create("https://ge.apache.org").toURL
+  private val PekkoProjectId        = ProjectId("pekko")
```

Comment:
What's the rationale for this? Do we usually explicitly set the id used with the Project Level Access Control (PLAC) feature even if PLAC isn't enabled? (or is PLAC enabled on the Apache instance?)

I'm also assuming that we are OK that different OSS pekko projects (for instance, pekko and pekko-http) share the same project id. Just making sure this was an intentional decision and there is a rationale for it.

Reply:
Yes, having the same id is intentional. It's a logical id that should give access to all OSS pekko projects. Everyone who is authorised to see the build scans from pekko core is also authorised to the build scans from pekko http, etc. 

@clayburn Could you please share more info about PLAC on the Apache instance and the approach to the project ids there?

Reply:
For now, this would be a no-op since ge.apache.org does not enable PLAC. But we have future plans to use it, so it would be good to go ahead and set it now. We intend to start setting this on current projects as well where we have not touched configuration recently.

Reply:
Thanks for the explanation. Sounds good and will resolve this comment.

Reply:
I can't mark this resolved but it can be considered so.